components/gc-subway/_base.scss

```diff
@@ -3,134 +3,157 @@
-					padding-bottom: 0;
-				}
-			}
+			margin-bottom: 0;
+			margin-left: -.685em;
+			padding-left: .5em;
 
-			dd {
-				border-left: 4px solid #26374a;
-				line-height: inherit;
-				margin-bottom: 0px;
-				margin-top: -3px;
-				padding: 0px 20px 20px 1em;
+			li {
+				border-left: 4px solid $accent-blue;
+				line-height: 1.25em;
+				padding-bottom: 1.25em;
+				padding-left: 1em;
+				position: relative;
 
 				&:last-child {
-					border-left: 5px solid transparent;
+					border-left-color: transparent;
 					padding-bottom: 0;
 				}
-			}
 
-			li {
-				@include dt-li(30px, .05em, 1.2em, 1.4em, -.05em, 1.4em);
-				&::first-line {
-					line-height: 1;
-				}
-				&:last-child {
-					border-bottom: 4px solid #26374a;
-					border-bottom-left-radius: 6px;
-					border-left: 4px solid #26374a;
+				a {
+					display: inline-block;
+
+					&::before {
+						@include subway-dot;
+					}
+
+					&.active {
+						color: #333;
+						cursor: default;
+						text-decoration: none;
+
+						&::before {
+							box-shadow: 0 0 0 10px $accent-blue inset;
+						}
+
+						&:hover, &:focus {
+							color: #333;
+							text-decoration: none;
+						}
+					}
+
+					&:not(.active):hover::before,
+					&:not(.active):focus::before {
+						box-shadow: 0 0 0 4px #fff inset;
+					}
 				}
 
 				ul {
-					margin-top: 20px;
-					padding-left: .55em;
+					margin: 1em 0 0;
+
+					li {
+						&:last-child {
+							padding-bottom: 0;
+						}
+					}
 
-					li:last-child {
-						border-bottom-width: 0px;
-						padding-bottom: 0px;
+					&.noline {
+						li {
+							border-left-color: transparent;
+
+							&::before {
+								display: none;
+							}
+
+							a.active::after {
+								background-color: $accent-blue;
+								content: "";
+								display: block;
+								height: 4px;
+								left: -1.75em;
+								position: absolute;
+								top: .5em;
+								width: 1.125em;
+							}
+						}
 					}
 				}
 			}
 		}
+	}
+	&.gc-subway-index {
+		h2 {
+			position: static;
+		}
+
+		dl {
+			margin-left: .5em;
+
+			dt, dd {
+				border-left: 4px solid $accent-blue;
+				font-weight: normal;
+				margin: 0;
+				padding-left: 1em;
+				position: relative;
+
+				&:last-of-type {
+					border-left-color: transparent;
+					padding-bottom: 0;
+				}
+			}
+
+			dt {
+				a::before {
+					@include subway-dot;
+				}
+			}
 
-		h1 {
-			float: left;
+			dd {
+				padding-bottom: 1.25em;
+				padding-top: .25em;
+			}
 		}
 	}
+}
 
-	/* Double H1 */
-	&.gc-subway h1,
-	&.gc-subway-section .gc-subway-h1 {
-		background-color: #fff;
-		border-bottom-width: 0px;
-		color: #555;
-		font-size: 1.3em;
-		margin-right: 20px;
-		margin-top: -19px;
-		padding: 0px 20px 10px 0px;
+.gc-subway-section {
+	hgroup {
+		p {
+			display: none;
+		}
 	}
 }
 
-// Remove the border for index pages, when an h1 precede the subway nav.
-h1 ~ .provisional.gc-subway {
-	border-right: none;
-	border-top: none;
+.gc-subway-support {
+	margin-top: 3em;
```

Comment:
Can this be removed, or the amount shrunk significantly?  Adds a huge amount of space between menu and related information. 
![image](https://user-images.githubusercontent.com/9367162/218215983-9b60a775-2a05-4f6f-89f7-4301e2af412b.png)

Reply:
Can we add a border, or something else?  the padding looks like it's broken.  Also, per my comments below, there was a script that moved the dom order so that was never an issue

Reply:
Hi @Christopher-O, we had consensus at PP that a reasonable gap was enough to break the sequence while not looking odd in terms of usability. I agree that we could look into shrinking a bit that gap, but all in all for performances reasons, as well as simplicity and accessibility, we decided that the gap is the way to go unless proven otherwise through something like usability testing.

Reply:
May be we can explore a few alternative, like Chad proposed to have a smaller gap or use a top border (hr section separator)

Like proposing a few prototype for discussion on alternative or to keep this one with the 3em space in between.

@GormFrank how much performance hit this will create? Like if we duplicate the "related links" in the right column and use CSS visibility to show/hide it in the different view ports.